pkg/cmd/roachtest/backup.go

```diff
@@ -52,9 +52,15 @@ func registerBackup(r *testRegistry) {
 		c.Run(ctx, c.All(), `./workload csv-server --port=8081 &> logs/workload-csv-server.log < /dev/null &`)
 		time.Sleep(time.Second) // wait for csv server to open listener
 
-		c.Run(ctx, c.Node(1), "./workload", "fixtures", "import", "bank",
+		importArgs := []string{
+			"./workload", "fixtures", "import", "bank",
```

Comment:
this is `bank` not `tpcc` and bank doesn't accept the `deprecated-fk-indexes` flag.